eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -60,7 +61,7 @@ stages:
                   deploy:
                     steps:
                       - checkout: self
-                      - script: |
+                      - pwsh: |
                           export DETECTED_PACKAGE_NAME=`ls $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}-*.tgz`
```

Comment:
Does this work? pwsh doesn't know what export is. We should either switch this back to script or update it like you did below in pwsh.

Reply:
oh yes i didn't intend to change anything here